### DISCUSSION TITLE
How to get bottom corner rounded?

### DISCUSSION THREAD

**Discussion by HarshitSingh16** [paulmcauley/klassy#112] [Q&A] [answered]
hello

I have checked the option to make bottom window corners round but it dosen't work pls help

**Comment by paulmcauley** [comment, marked as answer]
The best option is to increase the size of the borders in the KDE window decoration settings menu and not check "Round bottom corners when no window borders" at all.

If you want corners clipped (why you would want to do this, I don't know!) then you will need to also install a KWin desktop effect such as https://github.com/a-parhom/LightlyShaders , but this does not work as well as it used to any more due to changes in KWin.

**Comment by paulmcauley** [comment]
See also discussion at: https://github.com/paulmcauley/klassy/issues/61

**Reply by HarshitSingh16** [reply]
thank you sir

**Reply by HarshitSingh16** [reply]
recently updated to kde plasma 6 and klassy is working just fine !

klassy is awsome

**Comment by paulmcauley** [comment]
Windows without borders will now also be able to be rounded with the upcoming 6.5 release for Plasma 6.5
